app.json

```diff
@@ -4,7 +4,7 @@
   "environments": {
     "test": {
       "scripts": {
-        "test": "bundle exec rake"
+        "test": "bundle exec rake; bundle exec standardrb"
```

Comment:
It would be better to have `bundle exec rake` also run standardrb. 

See: https://github.com/testdouble/standard#using-with-rake

Reply:
Fixed :)